app/views/content/train-to-be-a-teacher/qualifications-you-need-to-teach.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "What qualifications do I need to be a teacher?"
 heading: "What qualifications do I need to be a teacher?"
-subcategory: Qualifications and experience
+subcategory: Qualifications
 description: |-
   Find out what qualifications you need to be a teacher in primary and secondary schools in England and which subjects you can train to teach.
-navigation: 5.30
+navigation: 20.20
```

Comment:
Would be worth adding in a specific `navigation_title:` (as per GSE PR) to control what appears on the nav card - presumably we don't want a question here?

Reply:
Will add a nav card title of: Qualifications you need to be a teacher